source/math/FloatVectorList.ooc

```diff
@@ -212,4 +212,33 @@ FloatVectorList: class extends VectorList<Float> {
 		previous free()
 		result
 	}
+	median: func -> Float {
+		result: Float
+		tempVector := this copy()
+		tempVector sort()
+		if(Int odd(this count))
```

Comment:
There should be a space between `if` and `(`. Our style checker will fix things like these, but for your own sake, make a habit out of being consistent.